integrations/fathom/src/fathomScript.raw.js

```diff
@@ -5,9 +5,28 @@
     element.setAttribute('data-site', siteId);
     element.setAttribute('data-spa', 'auto');
     element.src = 'https://cdn.usefathom.com/script.js';
+    const GRANTED_COOKIE = '__gitbook_cookie_granted';
+
+    function getCookie(cname) {
+        const name = `${cname}=`;
+        const decodedCookie = decodeURIComponent(document.cookie);
+        const ca = decodedCookie.split(';');
+        for (let i = 0; i < ca.length; i++) {
+            let c = ca[i];
+            while (c.charAt(0) === ' ') {
+                c = c.substring(1);
+            }
+            if (c.indexOf(name) === 0) {
+                return c.substring(name.length, c.length);
+            }
+        }
+        return '';
+    }
```

Comment:
No clue if it's possible at all, but do we have any way to move this function to a common .js file we may import and reuse in all of these?

Reply:
I’m not sure - because each time it’s published it bundles the code I think it’s better to have them duplicate code (also because people might see these as examples, and copy the files in a specific integration so maybe better to have everything encapsulated in one)